samples/matter/light_bulb/README.rst

```diff
@@ -119,6 +122,69 @@ Factory data support
     :start-after: matter_door_lock_sample_factory_data_start
     :end-before: matter_door_lock_sample_factory_data_end
 
+.. _matter_light_bulb_aws_iot:
+
+AWS IoT integration
+===================
+
+The sample can be configured to communicate with `AWS IoT Core`_ to control attributes in supported clusters on the device.
+After a connection has been established, the sample will mirror these attributes in the AWS IoT shadow document.
+This makes it possible to remotely control the device using the `AWS IoT Device Shadow Service`_.
+The supported attributes are ``OnOff`` from the ``OnOff`` cluster and ``CurrentLevel`` from the ``LevelControl`` cluster.
+
+The following figure illustrates the relationship between the AWS IoT integration layer and the light bulb sample:
+
+.. figure:: /images/aws_matter_integration.svg
+   :alt: Sample implementation of the AWS IoT integration layer
+
+   AWS IoT integration layer implementation diagram
+
+The following figure illustrates the interaction with the AWS IoT shadow Service:
+
+
+.. figure:: /images/aws_matter_interaction.svg
+   :alt: Interaction with the AWS IoT shadow service
+
+   AWS IoT Shadow and Matter interaction diagram
+
+AWS IoT setup and configuration
+-------------------------------
+
+To set up an AWS IoT instance and configure the sample, complete the following steps:
+
+1. Complete the setup and configuration described in the :ref:`lib_aws_iot` documentation to get the host name, device ID, and certificates used in the connection.
+#. Set the :kconfig:option:`CONFIG_AWS_IOT_BROKER_HOST_NAME` and :kconfig:option:`CONFIG_AWS_IOT_CLIENT_ID_STATIC` options in the :file:`overlay-aws-iot-integration.conf` file.
+#. Import the certificates mentioned in the :file:`light_bulb/src/aws_iot_integration/certs` file.
+#. Build the sample using the following command:
+
+   .. code-block:: console
+
+      west build -p -b nrf7002dk_nrf5340_cpuapp -- -DOVERLAY_CONFIG="overlay-aws-iot-integration.conf"
+
+#. Flash the firmware and boot the sample.
+#. |connect_kit|
```

Comment:
Should these steps be switched? I don't think you can flash the sample without connecting the kit.

Reply:
Quick search shows that this order is used in _most_ cases so it looks like a general issue.

Reply:
Alright, I'll look into this as a general improvement that can be implemented outside of this PR, then.